src/components/report-viewer.tsx

```diff
@@ -111,14 +113,22 @@ export const ReportViewer = ({ report, onEditClick }: Props) => {
       </div>
 
       {report.comment && (
-        <CommentCard
-          report={report}
-          showReporter={false}
-          linkToReport={false}
-        />
+        <div
+          class={css`
+            max-width: 15rem;
+          `}
```

Comment:
@varCepheid Can we apply the max-width directly to the `<Card>` component that contains the `<ReportViewer>` [here](https://github.com/Pigmice2733/peregrine-frontend/blob/dev/src/routes/report.tsx#L66)

That way if the field names happen to be longer then the comment will still be the full width of the report viewer card, which would fix this:

<img width="1212" alt="Screen Shot 2022-01-21 at 8 56 47 PM" src="https://user-images.githubusercontent.com/13206945/150625300-acd94b2f-05b3-4173-9276-f306c580eb76.png">

Also, I think a little wider than 15rem might be good so it does not feel quite so squished, maybe 20rem or 25rem?